S3M Doc updates (#1970)
SumoLogic/sumologic-documentation: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md

Here are some best practices for configuring the Sumo Logic Distribution for OpenTelemetry.

## Configuration Location & Structure
The Sumo Logic OpenTelemetry Collector configuration is comprised of two parts. Based on the platform, the configuration by default is stored at the following location

```
Linux or MacOS : /etc/otelcol-sumo/
Windows : C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config 
```

:::note
If you did a manual install of the collector, your configuration might be in a different location.
:::

* **Sumo Logic Logic-defined configuration**. This is required by collector to properly communicate with SumoLogic SaaS service, and transmit data. All Sumo Logic preconfigured components are stored in `sumologic.yaml` file. This file is managed by the installation script and should never be changed manually. Depending upon your platform.
* **Data Source configuration**. Here, you can define all the configuration that tells the collector what data to collect, how to process it (including adding metadata), and send it to Sumo Logic. All the user-defined configuration resides under `conf.d` directory.

> Any configuration for a Source (MySQL, Nginx, Application Logs, etc) should be stored in a separate file with descriptive name under the `conf.d` directory. For example, a file named `conf.d/mysql.yaml` can contain configuration to collect MySQL data (Logs & Metrics), and will contain the necessary receiver, processors and the pipeline that together inform the collector on how to collect and send the MySQL data.

Reviewer: ```suggestion
Any configuration for a Source (MySQL, Nginx, Application Logs, etc) should be stored in a separate file with descriptive name under the `conf.d` directory. For example, a file named `conf.d/mysql.yaml` can contain configuration to collect MySQL data (logs and metrics), and will contain the necessary receiver, processors and the pipeline that together inform the collector on how to collect and send the MySQL data.
```

Here are some best practices for configuring the Sumo Logic Distribution for OpenTelemetry.

## Configuration Location & Structure
The Sumo Logic OpenTelemetry Collector configuration is comprised of two parts. Based on the platform, the configuration by default is stored at the following location

```
Linux or MacOS : /etc/otelcol-sumo/
Windows : C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config 
```

:::note
If you did a manual install of the collector, your configuration might be in a different location.
:::

* **Sumo Logic Logic-defined configuration**. This is required by collector to properly communicate with SumoLogic SaaS service, and transmit data. All Sumo Logic preconfigured components are stored in `sumologic.yaml` file. This file is managed by the installation script and should never be changed manually. Depending upon your platform.

> * **Data Source configuration**.
> Here you can define all the configuration that tells the collector what data to collect, how to process it (including adding metadata), and send it to SumoLogic. All the user defined configuration resides under `conf.d` directory.

Reviewer: ```suggestion
* **Data Source configuration**. Here, you can define all the configuration that tells the collector what data to collect, how to process it (including adding metadata), and send it to Sumo Logic. All the user-defined configuration resides under `conf.d` directory.
```

Here are some best practices for configuring the Sumo Logic Distribution for OpenTelemetry.

> ## Configuration Location & Structure

Reviewer: ```suggestion
## Configuration location and structure
```

> Sumo Logic OpenTelemetry Collector configuration is comprised of two parts. Based on the platfrom the configuration by default is stored at the following location

Reviewer: ```suggestion
The Sumo Logic OpenTelemetry Collector configuration is comprised of two parts. Based on the platform, the configuration by default is stored at the following location
```

```
Linux or MacOS : /etc/otelcol-sumo/
Windows : C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config 
```

> :::note
> If you did a manual install of the collector, then the configuration might be in a different location.
> :::
> 
> ### 1. Sumo defined configuration 
> This is required by collector to properly communicate with SumoLogic SaaS service, and tranmit data. All Sumo Logic preconfigured components are stored in `sumologic.yaml` file. This file is managed by the installation script and should never be changed manually. Depending upon your platform 
> 
> ### 2. Data Source configuration

Reviewer: ```suggestion

:::note
If you did a manual install of the collector, your configuration might be in a different location.
:::

* **Sumo Logic Logic-defined configuration**. This is required by collector to properly communicate with SumoLogic SaaS service, and transmit data. All Sumo Logic preconfigured components are stored in `sumologic.yaml` file. This file is managed by the installation script and should never be changed manually. Depending upon your platform.
* **Data Source configuration**.
```

Here you can define all the configuration that tells the collector what data to collect, how to process it (including adding metadata), and send it to SumoLogic. All the user defined configuration resides under `conf.d` directory.

> Any configuration for a Source (MySQL, Nginx, Application Logs, etc) should be stored in a seperate file with descriptive name under the `conf.d` directory. For example, a file named `conf.d/mysql.yaml` can contain configuration to collect MySQL data (Logs & Metrics), and will contain the necessary receiver, processors and the pipeline that together inform the collector on how to collect and send the MySQL data.

Reviewer: ```suggestion
Any configuration for a Source (MySQL, Nginx, Application Logs, etc) should be stored in a separate file with descriptive name under the `conf.d` directory. For example, a file named `conf.d/mysql.yaml` can contain configuration to collect MySQL data (Logs & Metrics), and will contain the necessary receiver, processors and the pipeline that together inform the collector on how to collect and send the MySQL data.
```
